bin/informational/patron_information

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+"""Find information about a particular patron based on barcode and pin.
+"""
+
+import os
+import sys
+
+bin_dir = os.path.split(__file__)[0]
+package_dir = os.path.join(bin_dir, "..", "..")
+sys.path.append(os.path.abspath(package_dir))
+
+from core.scripts import Script
+import argparse
+from core.model import Library
+from api.authenticator import LibraryAuthenticator, PatronData
+
+class PatronInformationScript(Script):
+    @classmethod
+    def arg_parser(cls):
+        parser = argparse.ArgumentParser()
+        parser.add_argument(
+            '--library-id',
+            help="ID of the library the patron belongs to.",
+            required=True,
```

Comment:
You can subclass `core.scripts.LibraryInputScript` to get code that takes a library name from the command line and turns it into a `Library` object.

See `circulation/scripts.py:NovelistSnapshotScript` for an example. `parse_command_line` will put a list of libraries in `parsed_libraries` and you can either pick the first one or iterate over the list.